test/e2e/common/configmap.go

```diff
@@ -30,9 +30,9 @@ var _ = Describe("[sig-api-machinery] ConfigMap", func() {
 	f := framework.NewDefaultFramework("configmap")
 
 	/*
-		    Testname: configmap-in-env-field
-		    Description: Make sure config map value can be used as an environment
-			variable in the container (on container.env field)
+		Release : v1.9
+		Testname: ConfigMap from environment field
```

Comment:
based on formatting of your other test cases I would expect this to be "ConfigMap, from environment field"

Reply:
I agree. Fixed.